numpy/core/src/multiarray/convert_datatype.h

```diff
@@ -57,7 +57,7 @@ PyArray_FindConcatenationDescriptor(
         npy_intp n, PyArrayObject **arrays, PyObject *requested_dtype);
 
 NPY_NO_EXPORT int
-PyArray_AddCastingImplmentation(PyBoundArrayMethodObject *meth);
+PyArray_AddCastingimpelmentation(PyBoundArrayMethodObject *meth);
```

Comment:
Same here. You can fix spellign w/o changing the capitalization.

Reply:
It is still misspelled, should be "Implementation".

Reply:
done. Thank you for reviewing